websites/D/discord.style/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "bris",
+    "id": "656165298818514946"
+  },
+  "service": "Discord Style",
```

Comment:
Oh and i noticed it when I was clicking Request Changes button, SERVICE name must be the same as FOLDER name